server/api/json.js

```diff
@@ -55,7 +55,7 @@ exports.register = function (server, options, next) {
 
   server.route({
     method: 'GET',
-    path: '/api/json/pages/{slug}/{revision}',
+    path: '/api/pages/{slug}/{revision}.json',
```

Comment:
There should be no `/api/pages/` prefix. The URL should be `/{slug}/{revision}.json`. 

E.g.

* for `https://jsperf.com/slugs` the URL would be `https://jsperf.com/slugs.json`
* for `https://jsperf.com/slugs/3` the URL would be `https://jsperf.com/slugs/3.json`

This matches the `.atom` feeds for tests.

Sorry if my earlier comment wasn’t clear!